src/engine/currencyEngine.js

```diff
@@ -378,9 +378,14 @@ export class CurrencyEngine {
     }
     const customFeeSetting = this.engineInfo.customFeeSettings[0]
     const customFeeAmount = customNetworkFee[customFeeSetting] || '0'
-    if (networkFeeOption === 'custom' && customFeeAmount !== '0') {
+    if (networkFeeOption === 'custom') {
+      const customFeeNumber = Number(customFeeAmount)
+      if (customFeeNumber < 1 || isNaN(customFeeNumber)) {
+        // TODO: Throw proper InvalidCustomFee error
+        throw new Error(`1 satPerByte`)
```

Comment:
This does not look like a clean error. Instead create an error object and set `e.name` to `ErrorBelowMinimumFee`